enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java

```diff
@@ -146,6 +151,10 @@ public Response operationApprove(
         approveRequest.setUserId(userId);
         approveRequest.setSignatureType(SignatureType.enumFromString(signatureFactors.name())); // 'toString' would perform additional toLowerCase() call
         approveRequest.setApplicationId(applicationId);
+        // Prepare additional data
+        approveRequest.getAdditionalData().put("activation_id", activationId);
```

Comment:
Perhaps it would be better to define constants for these 3 strings used in additional data.

Reply:
Fixed